src/components/OSCALControlProse.js

```diff
@@ -420,6 +419,10 @@ export function OSCALReplacedProseWithByComponentParameterValue(props) {
         setParameter.values ??= [null];
         const setParameterRef = setParametersRefs[segment];
         // TODO - support for more than 1 item in values arrays
```

Comment:
If you have the familiarity with the surrounding code to do so (and because this ticket rolls up a general tech investments epic), can you convert this TODO to a GitHub issue (labeled as an enhancement) while you're in here?